api/cpp/include/slint-testing.h

```diff
@@ -88,6 +88,54 @@ class ElementHandle
         return std::nullopt;
     }
 
+    /// Returns the accessible-value-maximum of that element, if any.
+    std::optional<float> accessible_value_maximum() const
+    {
+        if (auto item = private_api::upgrade_item_weak(inner)) {
+            SharedString result;
+            if (item->item_tree.vtable()->accessible_string_property(
+                        item->item_tree.borrow(), item->index,
+                        cbindgen_private::AccessibleStringProperty::ValueMaximum, &result)) {
+                float value = 0.0;
+                cbindgen_private::slint_string_to_float(&result, &value);
```

Comment:
If this function fails, wouldn't it be better to return `std::nullopt` instead of `{0.0}`? It seems to me that this is also how the Rust function works that you implemented.

Reply:
Oh yep indeed. I will fix it now